src/Avalonia.Controls.DataGrid/Automation/Peers/DataGridCellAutomationPeer.cs

```diff
@@ -0,0 +1,22 @@
+using Avalonia.Automation.Peers;
+
+namespace Avalonia.Controls.Automation.Peers;
+
+public class DataGridCellAutomationPeer : ContentControlAutomationPeer
+{
+    public DataGridCellAutomationPeer(DataGridCell owner)
+        : base(owner)
+    {
+    }
+
+    public new DataGridCell Owner => (DataGridCell)base.Owner;
+
+    protected override AutomationControlType GetAutomationControlTypeCore()
+    {
+        return AutomationControlType.Custom;
```

Comment:
UWP implementation uses three state possible values here:
https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/main/Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/Automation/DataGridCellAutomationPeer.cs#L82-L98

Reply:
But I guess it's not as necessary, since basic text works either way.